### PR TITLE
Corrected the alignment of client project page content

### PR DESCRIPTION
**Pull request by Priyanka-Kaintura**
Targets #2983

### Description
Corrected the alignment of client project page content.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by Priyanka-Kaintura** [comment]
@GauravGusain98 
Sir, I have checked the content alignment for almost all range of screen sizes and especially for edge cases with screen size xs, sm, md, l, xl as suggested. But if there is still a problem then please let me now.

File: Modules/Client/Resources/views/subviews/edit-client-projects.blade.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="card">
+<div class="overflow-x-scroll card" style="min-width:660px;">
```
Comment (Ayush8923) [inline Modules/Client/Resources/views/subviews/edit-client-projects.blade.php:1]: Please sue bootstrap classes instead of inline-css.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]

File: Modules/Client/Resources/views/subviews/edit-client-projects.blade.php (hunk 1 above)
Comment (Priyanka-Kaintura) [reply Modules/Client/Resources/views/subviews/edit-client-projects.blade.php:1]: @Ayush8923 Sir I didn't find any specific bootstrap class to set min-width and so used inline style. I even went through the documentation but didn't find any class of this sort.

File: resources/sass/_variables.scss [hunk 2]
```diff
@@ -352,3 +353,7 @@ $grid-breakpoints: (
     xl: 1200px,
     hd: 1440px
 );
+
+$min-width: (
+    '660': 660px,
+);
```
Comment (Ayush8923) [inline resources/sass/_variables.scss:359]: I don't think, we need this. Please remove this. Then test it locally. And let me know, it's working fine or not.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
One comment else looks good to me.

**Review by Ayush8923 — APPROVED** [review]
Looks good to me.

**Pull request merged**